eos/utils/kinematic.cc

```diff
@@ -153,39 +174,86 @@ namespace eos
             result += ", " + i->first + '=' + stringify(_imp->variables_data[i->second]);
         }
 
+        // TODO: show aliases
+
         return result;
     }
 
+    void
+    Kinematics::alias(const std::string & alias, const std::string & name)
+    {
+        const auto i(_imp->variables_map.find(name));
+        const auto j(_imp->alias_map.find(name));
+        bool name_exists  = (_imp->variables_map.end() != i);
+        bool alias_exists = (_imp->alias_map.end() != j);
+
+        if (! name_exists)
+            throw UnknownKinematicVariableError(name);
+
+        if (alias_exists)
+        {
+            // TODO: proper error
```

Comment:
Can you please create a ``DuplicateKinematicAliasError`` in the same place where ``UnknownKinematicVariableError`` lives, and use it here? I was lazy.